### PR TITLE
Fix repository references

### PR DESCRIPTION
**Pull request by buildbreaker**

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ git checkout -b cool_new_feature
 ```
 
 Make your changes, then ensure that `make test` still passes. (Note: If you're
-changing `protoc-gen-connect-kotlin`, you should also run
+changing `protoc-gen-buf-gradle-plugin`, you should also run
 `make buildplugin` and `make generate` to
```
Comment (rebello95) [inline .github/CONTRIBUTING.md:40]: Invalid

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -2,19 +2,18 @@ name: ci
```
Comment (rebello95) [inline .github/workflows/ci.yml:2]:
```suggestion
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -2,19 +2,18 @@ name: ci
...
-    branches: [ main ]
+    branches:
+      - main
   pull_request:
-    branches: [ main ]
-  workflow_dispatch:
-
+  workflow_dispatch: {}
+permissions:
+  checks: read
```
Comment (rebello95) [inline .github/workflows/ci.yml:10]:
```suggestion
  contents: read
```

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -2,19 +2,18 @@ name: ci
...
 on:
   push:
-    branches: [ main ]
+    branches:
+      - main
   pull_request:
-    branches: [ main ]
-  workflow_dispatch:
-
+  workflow_dispatch: {}
```
Comment (rebello95) [inline .github/workflows/ci.yml:8]:
```suggestion
  workflow_dispatch: {} # support manual runs
```
nit for parity

File: .github/CONTRIBUTING.md [hunk 5]
```diff
@@ -18,9 +18,9 @@ you to accept the CLA when you open your pull request.
 [Fork][fork], then clone the repository:
 
 ```
-git clone git@github.com:your_github_username/connect-kotlin.git
-cd connect-kotlin
-git remote add upstream https://github.com/bufbuild/connect-kotlin.git
+git clone git@github.com:your_github_username/buf-gradle-plugin.git
+cd buf-gradle-plugin
+git remote add upstream https://github.com/bufbuild/buf-gradle-plugin.git
 git fetch upstream
```
Comment (rebello95) [inline .github/CONTRIBUTING.md:24]: "We'd love your help making Connect better!"

is also wrong above

Comment (buildbreaker) [reply .github/CONTRIBUTING.md:24]: > We'd love your help making the plugin better!

**Review by rebello95 — DISMISSED** [review]

**Review by rebello95 — APPROVED** [review]

**Review by rebello95 — APPROVED** [review]

**Pull request merged**